docs/source/intro.rst

```diff
@@ -150,10 +149,10 @@ Interacting with the Monte REPL
 Monte has a traditional "Read - Evaluate - Print Loop", or REPL, for
 exploration. Invoke it as `mt repl`. For example::
 
-  >>> 1 + 1
+  ▲> 1 + 1
```

Comment:
I couldn't get doctest to grok anything other than `>>>`. Unless you can help me figure that out, please don't do this. Keeping the examples easily tested is pretty important.